v2/tools/generator/internal/config/object_model_configuration.go

```diff
@@ -62,8 +62,15 @@ func NewObjectModelConfiguration() *ObjectModelConfiguration {
 	}
 
 	// Initialize group access fields here (alphabetical, please)
```

Comment:
Do we still need this comment here?

Reply:
I think so?

Reply:
Yes, as a guidepost for future maintainers that _here_ is where group access is configured.